tests/bulk_create/tests.py

```diff
@@ -39,6 +40,13 @@ def test_efficiency(self):
         with self.assertNumQueries(1):
             Country.objects.bulk_create(self.data)
 
+    @skipUnlessDBFeature('has_bulk_insert')
```

Comment:
This should work on each database that has bulk insert, but bug is specific for the Oracle.